src/branch-setup.ts

```
@@ -0,0 +1,68 @@
+			name: 'stage',
+			isUsed: true,
+		},
+		production: {
+			name: 'release',
+			isUsed: true,
+		},
+	}
+
+	constructor(octokit: Octokit, owner: string, repo: string) {
+		super(octokit, owner, repo)
+	}
```

Biome (1.9.4)

[error] This constructor is unnecessary.

Unsafe fix: Remove the unnecessary constructor.


(lint/complexity/noUselessConstructor)